Dockerfile.amd64

```diff
@@ -1,6 +1,11 @@
 FROM python:3-alpine
 MAINTAINER Martin <Munsio> Treml
 
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh
```

Comment:
Afaik we also need `ca-certificates` in order for git to function properly on pushes

Reply:
Potentially, I only tested with a ssh git repo which worked fine

Reply:
my hass is only available over https, so ca-certificates will probably be needed to call the api on that. Ill test both cases and get back to you